pkg/deb/docker-build-deb.sh

```diff
@@ -20,15 +20,10 @@
 
 set -ex
 
-ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../.. &> /dev/null && pwd )"
-IMAGE_NAME=apachepulsar/pulsar-build:debian-9-2.11
+ROOT_DIR=$(git rev-parse --show-toplevel)
 
-if [[ -z $BUILD_IMAGE ]]; then
-    # pull the image from DockerHub by default
-    docker pull $IMAGE_NAME
-else
-    docker build --platform linux/amd64 -t $IMAGE_NAME $ROOT_DIR/pulsar-client-cpp/pkg/deb
-fi
+IMAGE_NAME=apachepulsar/pulsar-build:debian-9-2.11-x86_64
```

Comment:
Back to the issue that has been discussed before, what's the process if I want to update the base image? How should I name the image?

Reply:
Ideally, we should use unique image tags. 

At this point the process would be to update the Dockerfile, build the image locally and push it to Dockerhub with the same tag. All the builds after that will sync and fetch the updated image.

Reply:
Got it. We should include this process when we added documents for the release process.